src/db/collections.ts

```diff
@@ -5,11 +5,9 @@ import {
   EventDoc,
   LevelDoc,
   NotificationDoc,
-  PersistentTaskDoc,
```

Comment:
No, these need to stay in.

This is because, the database in production still has data in these collections. Removing them means loosing that data. (and we want to keep for archival purposes)

Note that all other features that we removed (eg: Programs, Levels) also still have their respective collection definitions

Reply:
This applies to everything in this file

Reply:
Do we need to also put back TaskPayout and PersistentTaskStatus graphql types in this case? or remove this relation from PersistentTaskDoc and TaskDoc

Reply:
We don't really need types. We just need the collections to exist, so that the data won't get removed.

It's up to you if you want to put them back, but I suggest you don't bother.